.github/CHANGELOG.md

```diff
@@ -1,11 +1,32 @@
-# Release 0.23.0-dev
+# Release 0.24.0-dev
 
 ### New features since last release
 
+### Breaking changes
+
+### Improvements
+
+### Documentation
+
+### Bug fixes
+
+### Contributors
+
+This release contains contributions from (in alphabetical order):
+
+---
+
+# Release 0.23.0
+
+### New features since last release
+
+* Add `generate_samples()` to lightning.
+[(#247)](https://github.com/PennyLaneAI/pennylane-lightning/pull/247)
+
 * Add Lightning GBenchmark Suite.
 [(#249)](https://github.com/PennyLaneAI/pennylane-lightning/pull/249)
 
-* Add run-time and compile information in binary.
+* Support runtime and compile information.
```

Comment:
Were these carried over from the release candidate branch?

Reply:
Yes. I just edited some comments for clarification.